Sorry for the delayed response. Thanks for making those changes. I've fixed one of the failing CI checks. There's one more than requires a code change:

```
./tools/ansible/roles/dockerfile/defaults/main.yml
[70](https://github.com/ansible/awx/actions/runs/3036848463/jobs/5020943356#step:8:71)
  12:136    error    no new line character at the end of file  (new-line-at-end-of-file)
```

Once this is addressed we can merge this.

--- a/tools/ansible/roles/dockerfile/defaults/main.yml
+++ b/tools/ansible/roles/dockerfile/defaults/main.yml
@@ -9,5 +9,4 @@ template_dest: '_build'
 receptor_image: quay.io/ansible/receptor:devel
 
 # Helper vars to construct the proper download URL for the current architecture
-tini_architecture: '{{ { "x86_64": "amd64", "aarch64": "arm64", "armv7": "arm" }[ansible_facts.architecture] }}'
-kubectl_architecture: '{{ { "x86_64": "amd64", "aarch64": "arm64", "armv7": "arm" }[ansible_facts.architecture] }}'
+image_architecture: '{{ { "x86_64": "amd64", "aarch64": "arm64", "armv7": "arm", "ppc64le": "ppc64le" }[ansible_facts.architecture] }}'

--- a/tools/ansible/roles/dockerfile/templates/Dockerfile.j2
+++ b/tools/ansible/roles/dockerfile/templates/Dockerfile.j2
@@ -46,6 +46,13 @@ RUN dnf -y update && dnf install -y 'dnf-command(config-manager)' && \
 
 RUN pip3 install virtualenv build
 
+{% if image_architecture == 'ppc64le'  %}
+    RUN dnf -y update && dnf install -y wget && \
+    wget https://static.rust-lang.org/dist/rust-1.41.0-powerpc64le-unknown-linux-gnu.tar.gz && \
+    tar -zxvf rust-1.41.0-powerpc64le-unknown-linux-gnu.tar.gz && \
+    cd rust-1.41.0-powerpc64le-unknown-linux-gnu && \
+    sh install.sh ;
+{% endif %}
 
 # Install & build requirements
 ADD Makefile /tmp/Makefile

